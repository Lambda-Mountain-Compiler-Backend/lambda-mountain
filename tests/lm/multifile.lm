
main := (
   (print-s (dump-i a_counter))
   (print-s \n)

   (load-file (clone-rope A))
   (print-s (dump-i a_counter))
   (print-s \n)

   (load-file 'tests/data/file1.txt)
   (print-s (dump-i a_counter))
   (print-s \n)

   (load-file 'tests/data/5000.txt)
   (print-s (dump-i a_counter))
   (print-s \n)
);
