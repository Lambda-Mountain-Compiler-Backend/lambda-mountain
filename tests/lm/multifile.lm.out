0000000000000000
0000000000000002
0000000000000009
000000000000271c
